src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -115,6 +115,13 @@ public void clickCollapseExpandButton() {
         click(collapseExpandButton);
     }
 
+    public void expandPanels() {
+        if (isElementPresent("collapse-panels-button")) {
+            clickCollapseExpandButton();
+            waitForPanelsToExpand();
+        }
+    }
```

Comment:
This method might be useful in other UI tests involving results pages. Of course that should be a separate PR. Just saying.